### PR TITLE
EDU-12223 - Update Headless CMS widgets doc

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
#### Types of changes
- [ ] New content (guides, endpoints, app documentation)
- [x] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

To open the preview:

1. Open [this URL](https://developers.vtex.com/api/preview?branch=EDU-122223-hcms-widgets) to set up the portal with this branch changes.
2. Access the following URL: https://developers.vtex.com/docs/guides/faststore/headless-cms-widgets

### CODE REVIEW

File: docs/faststore/docs/headless-cms/widgets.mdx [hunk 1]
```diff
@@ -1,16 +1,315 @@
...
+      }
+    },
+
+  …
+
+  ```
+
+  </CH.Code>
+
+  </CH.Scrollycoding>
```
Comment (barbara-celi) [inline docs/faststore/docs/headless-cms/widgets.mdx:302]:
```suggestion
```

File: docs/faststore/docs/headless-cms/widgets.mdx [hunk 2]
```diff
@@ -1,16 +1,315 @@
...
+4. Open the terminal and run `yarn cms-sync` to synchronize your changes with the Headless CMS. Once the changes are synchronized, you will see the following message: `info: CMS synced successfully...`.
+5. Go to the VTEX Admin and access **Storefront > Headless CMS**.
+6. Open the page where you want to add the new section. In this context, we will add the `Custom Newsletter` to the `Home`.
+7. Add the section to the page, and you will notice the widget you have added, the [`media-gallery`](#media-gallery).
+
+    ![newsletter-hcms-interface](https://vtexhelp.vtexassets.com/assets/docs/src/scoped-widgets-one___6c8c5e28b43487efc6f6048afd8d8d15.gif)
+
+8. Click `Preview` to see the new component with the new image.
+
+    ![newsletter-store-interface](https://vtexhelp.vtexassets.com/assets/docs/src/scoped-widgets-two___9c76a69ce03639b33faa85eb2a7cb367.png)
```
Comment (barbara-celi) [inline docs/faststore/docs/headless-cms/widgets.mdx:315]:
```suggestion
    ![newsletter-store-interface](https://vtexhelp.vtexassets.com/assets/docs/src/scoped-widgets-two___9c76a69ce03639b33faa85eb2a7cb367.png)

</CH.Scrollycoding>
```

**Comment by barbara-celi** [comment]
Mari, fiz a sugestão de mudar o </CH.Scrollycoding> de lugar, colocando no final do documento. Simulei no [playground](https://play.codehike.org/#N4IgtgJgHiBcIFokB0B2AXAlugNgU1gAIByAdUwgHM90BnQgMwHsAnQgCTwEMJ9b6AwgFkAysTRIEaNOSo16XFnkKYMeFgy4BjZXnxg8GeqsLoAFsoDanHn0GiAugAoz6dAAdasAPTeIeADc9Jnd1WgA6APQ8KHCtJjA-Ji1ab0oAVwo8VM1aOnRWPG8LW2zaBC0wcqYglgDMPAB3AEpTMy50QgMWLXajQi0uVEJ02mUCwkM+nUI8wtm8LSwmVAjCABULMYGhlVQtHHT-QggO5XdMLQBrMIAaE5YQiCZG4YNUUfvMMC5qEfccEweHdCEMIF1CvcLDgLqhKJMINhWPQtEozqDUOC8IjOvE1BhwtJUABJYbmTD0DJZe4ATyY6UIjUwOBwjFUEFgRIQhEssmodFBAS4zK4ACN8M4AMRMuR0BBcIUi8V4ZpEACCwxq6nqTUITAYk20ZkZFH5GPB2HoDHS+2WqEJqG5llJeRY6SWmBW9BWhDML1MTFBEHBXBNsoDoIWHpWUp4iLh8oQMv5CAKibmSgQY2jqFVhDVs2i7gQoppWaLkygXDAAOUPtU8RY7lYHVU8NDyZoewmodQuozymzdodaElksIfPkguFODF+CJACFgo0VApCKKWA0Df5aKjMO47XqDeZlJ2BeYOjthqLlN9a+9oiHMSMxhayRYONxeGVCMIRN31E0GYTBPQgADV1gAUQADXzCAwFUcJCAAMVYE4aBnbE9mYFgfkPMV6U6bhejDfl7gOS4rj1d9lCUWhQg9IJSK7VBqzwEdUAAHwnU0WLYwgBMIbiABFsj3A9PWGQShLQbjJHkhTFKQITCCUtTJBkrieQAA16RYrlFJgoGybSpT065DOM2hWmk2zbJEikAS4GkUQsCyjOyRg0LGfAPThPUJK9PYvNwtZNO4yxtO+X48AQdIASBfwWFMpxJWi6g4oS4EWBs+zCAAVSyiBjB+agrUeMBCDpdI2EBQZWR+XpVHYuzWs42SdNOaJU2+PAUslLrYqwAxctatrCAABUeeod0jerDFONhm3ceKQoWXysH80NBvNUxeqQ9qtMiiAWC4Bh0AAK3KTdelTGJ0H6k6zsu67LjMO6oHQVpuIAJTe0x7oRJE2BwkK8M29tnzyGkcDbQt5miT6kLajrIoMREuAQSguBZdQaX69HMEx7HcZYGlRrywrAR4b02B8xYBRiCkIZUUrPLQma8CYcqEh5IRsSJ5xXA8LxfCiGJoXccIwBpcXYniRJ0HSApNxx7xCcxrU6gaRokAAZgARgYMwLvYdZ0AAFnWBcmCYABWAAGNUwAAcQARQATlFAAmZoDqJAAeAR2HCERUSYFkaXieNKAAPiJMdx10tyDI82htKJYTHNnFy5pTyyBgSMAVhwGkXywkwcKqCN6aWDEAuHDYP2YFkXjh9LBz9Rp6C7tplGT-TLJM0F3FCRR6CZckaN9L87F-UQAI0bQCCJABCSxzNTqyEFUaIl50IW3E8HxvDlyXIkRrh+HkOIEm8K+xjoJIUm8Wgelf+JQggJNeLlcwlDwAAfWAQAdhAVwC2ds8AAA48Dez1h7fwAA2O2Bs8AGwNkgr2SDRQGygTeb2hDwiUEwAwZogdg7hAEEwfw8dUBoG0owq6PpN5D1oNIEA2gsBCmiMgOAhBgBoGknwrAuA8B8KIHwtUjF0QCHigkPhtwhHCJAOgGkoQJGED4YZCO3B6EgCUVJQSfD-CaHSDgdAmj0BujwIY2yfCzyaMEUYuyfDMiwEcfwvhrC058OUYJAAvmgAJhjGEZ3oagAO3hKHUNoVyFAETE6ECimzTK1MkrhLQFneiOc1yDBYOCCY8V0n1w7owCqVV6S1WSDjLoRpmpIU2MoFugImT+TKbQLuPd-SgRSTFNJiV1DaRHmPFgE9sBmBAh+Eo35+Dz3-LvQCy9OQRPXh3Heah954EPiLE+Z89BSzlg-G+Ct77Xyfs8F+b8tAfxCNiH+spyjMBqsAwBdtoEQFFFwA20CkE-IdhAKBIDDYWz1kgrgDskFaAtgbLg0DoV2wtuEdwcJyERNjoQQABWSAHg-pu-d1nFMGclVc1E6zHg-GeBQipZzKiQiILALJ7iNFoosIuC0Xxw1AmjAWoYSb4DJgTHlWMcb8vJsxdATLJkkRFS8egtcWZgn+OkuGqAmCdBLmXDu9BRTK1BDgWggZObc3KbzLl-MMY7OPr4c+ctb6K2VqwImOB1ZCq1jqXWCBDbG1NubK2Nt7ZO1dp7H2rRi4QHMexChIdYl4DoQwphBrhgEqKuoDhHcRA9Ccf4kR6jxFeJAK6Nsij-FaNUdgec+biRs2LS4hxv8s0uJUe4zxkiQDJvSamkAJbCBBOGL2sJgdonRpobG+JUhEljmSYNHqBhMmoCmjULIeScYLUUIQZaq1QbyrhqGWgbZ8AnHRIq4aLViSdApCofwtTQaIhyc5OGV7eDNVoPcQIhhOgnpfXqNgQwy67kMIoT0jILBKEqQyPsWEJj0UWKQsuu6GKkMuIe6Iu0T0minmBtgMy55-kLvidASFUJsBiNWWs9xOkvEGNsUM7hHjhqWMQegMQLinUPDtNCt6nJwb2gYBAsMEKPkLvI68PBqCNObhHVp7c2azC6b6HpH5tLTpPcMrgo9uBjPQ5M6e2Gfy4cWVslZaB17Kd6hsveQFtkuCPqLU+iMbWX3OREU5xyLnJFSNc25X8Hn8ieZgIIrzgwQDwXgJBDBRRaAdg7UUvzoEgO9iAnBEAcEMCQRAeLeAQHQOIaQtFaAg7DriREsJzDhimYMBwwaGoIDrF6g2lRoiK2toECuzEiga0NdzZovhha4QdeMSAFtAju0CTcZgDx9b83lbzd23tPbQmMMHTEkdcbHQJNHJOpTp1zpXQQDdd6iMHqZ2zs5BQwwBYqzaJeGV3cwYdAVVDNRsN-L6nhqBw74nmmSbbu0mTFHbu916U9Hbr1bqHdU+p8eWmpnKF03M-TmzLNGdQCZ7bL1zNLIPtZ3ZYt7MHIviRpzdqzmP1SJcjz79dx3O-pSrMmAoCvNwVoI24LoFcCQRbJBXODYW29vCkBdt0uhY+VoT5EBvbItRVGqhK2iQlcTQ8Z6u39sfUsfo-b6x7r1YG41vNrbNefSBirfro3VFdfzb1ygJvS1Deca1MbE3ZTdZAMD9HKvDt+MbXNkJ8a51ROW0ViQ63UBJO0hrYVpN8ZEgXaEFgai5qKEKYGQlPBv3rQZvXJmeRpMxRNZVM1PLLW2YczEYnhhvBKxVk6l1GMEBup1vrI2JszaW2trbR2zt3Ze19hCcN+BPteVbm0+EHS5OA8U+HvleMIejPGRh0CcP7ALMR8sokGKcV4pOKQhg6hDAzBvOgZlhg+46XbUS-qZ-sqtGPRPoVU+BUkovJ0RoFhhjVQGLDa4J+w935FdP9djwseaitI9IV46edcaEKeT4rMueaEoYRqeeJ+lg5qgs2OVqxQ+Otqpylejqas4e9eTQje3qLefq7egaXeIaveEaA+fSGUUBQyeqrS9A7+RSRU1EpcMBZUiB1UbAjU2mkaqylgk+f+ZMGOWyReeyeOMIBOVYROLmTmz8FONyVO3mtOYwQQqAryYoiCiwEAcWWgesDseAeADs0KFsvO0CDADA0CesEAgweAdsHsEuKKlAeWkSAeo6xWCaPowhkeHCGsLsIhNI2upuuuzuKBoYgRfhBiJaOaGiFu1iRaMRtag2k2RAduriIAzaaRpavhoqnutk3uC24S-uhWnhQe46G2cEMckYZ4EYu6Ksg4DMkkRIk4AogwwwkswGx+7+7QTEG8aIgmvY-YjRUYdoEhvg-gQQgIseEQWBd85OaQmQO43guQ+QhQ3gOqzIMcWYzRXoFQgxbYiYfYusQ4kk1+z4jQQwH6gYcYkYJxABdyceZcEw2AjSgYzIhwro6IvcHYv8YBN45cb4kYA4Yxkk9wN4rSfcSggAmARnaVikYHqjCcofjcq17374xSh5F4ytB1Ew6EAPHaRyJzCVQAByTQtA+AbgDBZxPoXyr41ESB2k5J3cVJe8KUws6BUxwQsxMhxOixVIKxaxA43grA2MqAFIVQCAJxlJNAe8rQrE3CTROYHEGKgAnIS4rrCBiojcAobDEri0moDkbujGhXzf7EkFBkkUlslDL3AtIKa3irDWLuh2jGDTyWACCHFbQEkjHzCGkTF+BvozFhB8mnICnLHZCrFXzrFKCbGZA4A7GGnlA6mtgJiYwnG7E5itCCkCFoAGxIQADyoQb+VSb25wjwF0Ge0cjpc0I6RucBz4rE9Q2MKGEwvSlQOQ0ZA4wyLcSUDoEuhARZx+vSSZ4QpWPZzIAhesSEpI3+o545bI+Adp7IJ+hp3RoG7+VxGA9RwYJ+Z4M5089pw+yS85iaE5B6JGNYS5J+Dxhp9wRJowlphALJsp1JyU9wSgOg-mnkXKP+6JQRgq-5keuJv8HEAkBWMugeUk8uPop5PozAWgowAAvHrBbLAAgiWoAGQEJaGRKirEFW+aFpCQL5NpLA1upuSgAAjpkEoBAGHHcuwvwpYA4HYvYgWnpD8CEWxWEYRY+bzCRXKZ2qxZkTuOJHaM7iIH6HHuhLuJuIFMMKDKBIaYxuujVM2GMORTrubq2kwKKJWUsJpRRXgNRZgLRZopYCIuWnmixSNqbrRk8VgNkFxZkZcCsM5fbmWmIs7sSHiIZdxdpaWrpfpZYskWNCovZUAQ0IxekbZR5a5fojFY2mFaEVZd5b5aFclf5fEa2pbn5WNHwoYOkGAKSWxNFTyHwpBKgNMXcnwjZUlclQVR8GAOZRVVVTydZcJZlabqYlwOYiFa2pVdVfEbFXZD7vVR5TjP1cNuNflWbtlaWrlRlV1SlV5fmmqBYuoIqd+YQAADJih6B5UeU9V9XO6DXtUFHLVzaZVXVjRjUNWeVNbTVdWWWrX66pVLVhVxF64LWJF9YfWzXHUWLO4uxdgvm7QiAIa1IFlWFhCrwXW3WdWZEBFBHuWZE8WtoRGEBRH5H-UeVqLzU9a-VW642ZG24jVsXZFO75rYlkzw1hU3WtR3WfUu5iRyXiX8K4XM3o2lqiSyX7js2I140BWE2bh-WC0iV4BmJA35o-SLB4DbWlkym7T2XFyunBQ8BCj7AzYzXzbk18K0b+baA0gTQRyXDBEc3k0rWPV8JTSG1aBlwm2f5lzZIHDCjdCHWhHC0FpE0e0245Gc33WU38jO6u7K5vSq5023Xk1M2zV4A-DMikgrToB7WQmo3cXvUDXx2siqBJ2EA0oHUk1ZXfUi1JHi1sWA1TV8IACapZkEWdkdhRZd3VJ2NIJVBgidysadDWGdpast1F2QnQ+FeAAA-L7V9c7jovgEMGPSzVLVNc6eIjrTHR5UPR3cnftTgF3Trj3Xwm3Y6bnfnZvYXZ1gTd7aLcTU3QNhXc7jXTVM+WxA3dJMvZkbQOkKKHzTeAuMrAUKgCnQXYlc9Q9cXSACIG-R-coDqm4D6IfTPfjcA4tZfd1ZLb1dLa2qA+-XuDeI-YEpfT1mA5g3gF-VA7-YlG2H-UfQA5lS9dbSA-g5uICZAz-XnaQ-5DA8fVpafQg5baWtffmjtSw5QOEEI9gwJM-WxfkhAFvVbcAwJW+bJvSAmeuDWRI3dugKPew57afZPXojPbw0QJoPqovclWIyovEICCwGBIBtcVI6WtzXwtQuY4QEKKrBgLA17VwzraboVcVaVS1SAEIMKPovcHwjtZgJQK4IoqWtIjoK4yAHVYA9434-HUE6WrDOEyFcE5wloDEyFfE1Q7PSg5XeAIEyI7rTrSIkCHkOg+AzY9Q8A1qdGYQNUwQ246fUFQzDPRFeoI5WVQHVzT3X0-dXYyALVqtYg+XazfzS0fmvzPwLnqMxWuMyffAz7RoxM3Pc7uwLbKdDSHDV2p4zg9w6bgYHM9QDY93a9aWrM7QDFL7SoqJWzdMxjWULc2s8s87h48tVfcgydfmpsEMFRO-qDDwbMHQ1MysISPs18yY3FelZQ183U2lW5W8xwys+fXcwNt43vWVRZSAAIKZQcHgKQAqDSAICnLVUs5i01X4-iz0PgMSwEKS+S3E5S0gxs4RQS-SyS2S-pKU0-dwwzaNbg6opU+gM0-Q3gJBCwI8GRRbQc4i386K002C4CVKzK608A+0wZWs-rYAd01Fec7YwM0c9vZc3wgs3mqyzw5M-JeES838Baxi5o2i6XSa2y4U87gWSEBEHy4cwc8c-a99YM-02a-44G069a3zbazM+Gyi86x86s1ayYj86g6WiIAkDQNpvCMytuY0I8HCEhBNFPdsNYnBtjIhL66I5S3wvFYa9I0i0E267Y+44m0241UVdizS5y0SyS8SywBKX9XkwiyAIk0xfY92wyzSH2wO8TUO8tcm+y81hO724oDO5Wz2gK9HbFYKxuy4mNcothREgJAOke4QKUZBeUVJJIAnJOiIDSPsGYPm5gAAF7jAfi9BDBcGTzGg2CzJL5EhIqDnFkn57wISsSsiKpujDDaQ0irsDBSm0D3taDDJQZIePsrAvvKAgsftwieTfsn6-s4aiCFla0n44dcGKCDhodPuvsQAgEMhMgsgLBvtfZD5wwnM3PUBEBRSoDMBEC4aIda3giv3ZNlDWiRxCPhDaQOh2xIQuyBhtkfgQQwRwRge7TaA6BzIABUWn9KhQDA+bnQGKhHemogOnDoSChZwHoE7gueL+u+YGjI1xO5hSH4t5ex9ohAs55IKIKwh2TKp4nxQYrn-cRFVprJglxKin-cWzs6DoICSEaou5SlHnEYNnMU9wiqm5QXqqWAMwoEdRvRCoygcY2I9wv5NNmJqUlXfsRI0k68MppFCAZgHZYhlmAZ+y0hRychd8rmZO7mr8lOn89ytOKwQCwCUK8KUCfOoooKFscWktWgYWSCDs83Z0ehG3eCOWZCRI2Wv4n+VE2kU0gQOsKHgYYwLHPpK4CszYfYubEyN5uoHcYFgkDX1pkX5YhQbXy8HXUhhyjmpOxOfXihg3yhw3NOv85Qh+TAryHsWgiW7OcPJhYKCCooesesmgcK+C3sXAICWgaPSCCXLhbh4FQ6oc4ckc0cbYscBiIAKQtAcAIAyKSg7qFQfngT6gAiWiUkPwLAJCqARA0C7gUAAA3NzwErT3iAwGE3AKAM9ngKSUVTeGMnAAY2MLcAWl3NQu4DSEQz-arzjOr5wg6qE6gFcAb4YxryeAYIz3hOoE6ggItDcCwCAAEgEkAA) do Code Hike pra ter uma ideia de como ficaria, mas acredito que com o preview daqui dá pra visualizar melhor.

File: docs/faststore/docs/headless-cms/widgets.mdx [hunk 3]
```diff
@@ -1,16 +1,316 @@
...
+        }
+      }
+    },
+
+  …
+
+  ```
+
+  </CH.Code>
+
```
Comment (Mariana-Caetano) [inline docs/faststore/docs/headless-cms/widgets.mdx:301]:
```suggestion

---
```

File: docs/faststore/docs/headless-cms/widgets.mdx [hunk 4]
```diff
@@ -1,16 +1,316 @@
...
+### Synchronize the changes with Headless CMS
+
+4. Open the terminal and run `yarn cms-sync` to synchronize your changes with the Headless CMS. Once the changes are synchronized, you will see the following message: `info: CMS synced successfully...`.
+5. Go to the VTEX Admin and access **Storefront > Headless CMS**.
+6. Open the page where you want to add the new section. In this context, we will add the `Custom Newsletter` to the `Home`.
+7. Add the section to the page, and you will notice the widget you have added, the [`media-gallery`](#media-gallery).
+
+    ![newsletter-hcms-interface](https://vtexhelp.vtexassets.com/assets/docs/src/scoped-widgets-one___6c8c5e28b43487efc6f6048afd8d8d15.gif)
+
+8. Click `Preview` to see the new component with the new image.
```
Comment (henriquessb) [inline docs/faststore/docs/headless-cms/widgets.mdx:313]: Os números da lista aqui estão corretos? Parece que estão continuando a contagem da lista da seção anterior, e não sei se deveria ser assim. Vou deixar a sugestão aqui. Outra opção seria remover o cabeçalho, e talvez ajustar o texto do item 4.

```suggestion
1. Open the terminal and run `yarn cms-sync` to synchronize your changes with the Headless CMS. Once the changes are synchronized, you will see the following message: `info: CMS synced successfully...`.
2. Go to the VTEX Admin and access **Storefront > Headless CMS**.
3. Open the page where you want to add the new section. In this context, we will add the `Custom Newsletter` to the `Home`.
4. Add the section to the page, and you will notice the widget you have added, the [`media-gallery`](#media-gallery).

    ![newsletter-hcms-interface](https://vtexhelp.vtexassets.com/assets/docs/src/scoped-widgets-one___6c8c5e28b43487efc6f6048afd8d8d15.gif)

5. Click `Preview` to see the new component with the new image.
```

File: docs/faststore/docs/headless-cms/widgets.mdx [hunk 5]
```diff
@@ -1,16 +1,316 @@
...
+
+> ℹ️ To create a new section, such as the `Custom Newsletter`, follow the instructions in the [Creating a new store section](https://developers.vtex.com/docs/guides/faststore/building-sections-creating-a-new-section) guide.
+
+1. Open your store project code in a code editor and navigate to the `cms/faststore` folder.
+2. Open the `sections.json` file.
+3. In the `sections.json` file, find the section where you want to add the widget. In the following `sections.json` file example, the new section, `Custom Newsletter`, receives the [`media-gallery`](#media-gallery) widget.
+
+  <CH.Code>
+
+  ```json sections.json mark[34:39]
```
Comment (henriquessb) [inline docs/faststore/docs/headless-cms/widgets.mdx:152]: Ajustando indentação

```suggestion
<CH.Code>

```json sections.json mark[34:39]
```

File: docs/faststore/docs/headless-cms/widgets.mdx [hunk 6]
```diff
@@ -1,16 +1,316 @@
...
+          }
+        }
+      }
+    },
+
+  …
+
+  ```
+
+  </CH.Code>
```
Comment (henriquessb) [inline docs/faststore/docs/headless-cms/widgets.mdx:300]: Ajustando identação. Lembrar de ajustar o fechamento do bloco de código (` ``` `) acima.

```suggestion
</CH.Code>
```

**Review by henriquessb — APPROVED** [review]

**Pull request merged**